build.gradle

```diff
@@ -19,6 +19,9 @@ allprojects {
         jcenter()
```

Comment:
I will be deprecated in the next Gradle version. Is there anything that needs jcenter?

Reply:
thx!